README.md

```diff
@@ -25,7 +26,7 @@ This Docker Compose includes the following components:
 
 ### Build
 
-If outside the MITRE network, the cypress-inferno image will need to be build locally. This can be done by cloning the [source code repository](https://git.codev.mitre.org/projects/ABA/repos/cypress-inferno/browse), changing into the root directory of the repository, and running:
+If outside the MITRE network, the cypress-inferno image will need to be built locally. This can be done by cloning the [source code repository](https://github.com/projecttacoma/deqm-test-client), changing into the root directory of the repository, and running:
```

Comment:
```suggestion
If outside the MITRE network, the deqm-test-client image will need to be built locally. This can be done by cloning the [source code repository](https://github.com/projecttacoma/deqm-test-client), changing into the root directory of the repository, and running:
```